modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -0,0 +1,1494 @@
+= Object Storage Properties 
+:description: Object Storage properties list. 
+
+Object storage properties are part of the xref:reference:properties/cluster-properties.adoc[Cluster Properties] group.
+
+For information on how to edit cluster properties, see xref:manage:cluster-maintenance/cluster-property-configuration.adoc[]. Some properties require the cluster to be restarted in order to be applied; see a specific property's reference for whether restart is required.
+
+== Cloud configuration
+
+Object storage properties should only be set if you enable xref:manage:tiered-storage.adoc[Tiered Storage]. 
+
+=== cloud_storage_access_key
+
+AWS or GCP access key. This access key is part of the credentials that Redpanda requires to authenticate with object storage services for Tiered Storage. This access key is used with the <<cloud_storage_secret_key>> to form the complete credentials required for authentication.
+
+To authenticate using IAM roles, see <<cloud_storage_credentials_source>>.
+
+*Requires restart:* No
+
+*Optional:* No
+
+*Visibility:* `user`
+
+*Type:* string
+
+*Default:* `null`
+
+---
+
+=== cloud_storage_api_endpoint
+
+Optional API endpoint.
+
+- AWS: when left blank, it's automatically generated using <<cloud_storage_region,region>> property, and <<cloud_storage_bucket,bucket>> property. Otherwise uses the value assigned.
+- GCP: use `storage.googleapis.com`.
```

Comment:
```suggestion
- AWS: When blank, this is automatically generated using <<cloud_storage_region,region>> and <<cloud_storage_bucket,bucket>>. Otherwise, this uses the value assigned.
- GCP: Uses `storage.googleapis.com`.
```